### PR TITLE
Responses-Shimel

### PR DESCRIPTION
**Pull request by eshimel**

### CODE REVIEW

**Comment by eshimel** [comment]
The quote is John F Kennedy

File: challenge.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@
 # Question 1: Ruby objects and classes.
 # How would one find the class of the below objects.
 
+puts whateveritis.class
```
Comment (MattB-GA-Boston) [inline challenge.rb]: Correct.

File: challenge.rb [hunk 2]
```diff
@@ -17,18 +18,25 @@
...
 
+true
+
 # 2.1 Substitue "hello" && 4.5 for 33 && 55 above. What's the output?
+true
 # 2.2 Substitue nil && 88 for 33 && 55 above. What's the output?
+false
 # 2.4 Substitue 63.4 < 64. What's the output?
-
+false
```
Comment (MattB-GA-Boston) [inline challenge.rb]: Nope, sorry - that last one is `true`.

File: challenge.rb [hunk 3]
```diff
@@ -17,18 +18,25 @@
...
 
 # How would you (Yes, you can look up the String methods!):
 
 # 3.1. determine if a string is included another another string?
+puts "simple string".include?('ple')
 # 3.2. determine if the length of a string?
+puts "simple string".length
 # 3.3. Go from the string "eels in my hovercraft" to "fish in my sink" but don't
 # change the string "eels in my hovercraft"
+puts "eels string" + 'fish string'
```
Comment (MattB-GA-Boston) [inline challenge.rb]: That doesn't produce the string we're looking for. We want to take the string about eels and transform it into the string about fish.

Here's one way we could do it.
`"eels in my hovercraft".gsub("eels","fish").gsub("hovercraft","sink")`

File: challenge.rb [hunk 4]
```diff
@@ -38,9 +46,12 @@
 vehicle = "4 wheel Bus"
 
 # 4.1 to "6 wheel Bus"
+puts vehicle.gsub!(/4/, '6')
 # 4.2 to "6 wheel tank"
+puts vehicle.gsub!(/Bus/, 'tank')
 # 4.3 Create a NEW string from the string in 4.2 that will be
 # "6 wheel tank that shoots rainbow bunnnies"
+var new_vehicle = vehicle + " that shoots rainbow bunnies"
```
Comment (MattB-GA-Boston) [inline challenge.rb]: No var - this is Ruby, not JavaScript!

File: challenge.rb [hunk 5]
```diff
@@ -51,6 +62,23 @@
...
+  guess = gets.chomp.to_i
+  unless tries < max_tries
+    puts "Sorry, that's wrong...but I'm flattered"
+    break
+  end
+  tries += 1
+end
+
+puts "You guessed right!!" if guess == number
+
```
Comment (MattB-GA-Boston) [inline challenge.rb]: Close, but you're missing Tom's special addition of shouting at the user if someone overestimates by 10 yrs or more. Do you have any ideas about how you might add that?

**Comment by MattB-GA-Boston** [comment]
3.75 of 5 (-0.25 on Q3 and Q4, -0.5 on Q5)

Nice work.

**Pull request closed without merging**